source/guides/namespace/extending-a-namespace-registration-period.rst

```diff
@@ -115,14 +115,14 @@ Once the **RegisterNamespaceTransaction** gets confirmed, double-check that the
     type:           Root namespace
     owner:          TBULEA...IPS4
     startHeight:    52000
-    endHeight:      54000
+    endHeight:      178100
```

Comment:
shouldn't the endHeight be 225800?
53000(original endHeight) + 172800 (extension) = 225800?

Reply:
well seen, I missed a 0